src/Take.Blip.Builder.UnitTests/Actions/ForwardToDeskActionTests.cs

```diff
@@ -0,0 +1,76 @@
+using System.Threading.Tasks;
+using NSubstitute;
+using NSubstitute.ReceivedExtensions;
+using Serilog;
+using Take.Blip.Builder.Actions.ForwardMessageToDesk;
+using Take.Blip.Builder.Actions.ForwardToDesk;
+using Xunit;
+
+namespace Take.Blip.Builder.UnitTests.Actions
+{
+    public class ForwardToDeskActionTests : ActionTestsBase
+    {
+        public ForwardToDeskActionTests()
+        {
+            ForwardToDesk = Substitute.For<IForwardToDesk>();
+            StateManager =Substitute.For<IStateManager>();
+            Logger = Substitute.For<ILogger>();
+            Settings = new ForwardToDeskSettings();
+        }
+        
+        public IForwardToDesk ForwardToDesk { get; }
+        public IStateManager StateManager { get; }
+        public ILogger Logger { get; }
+
+
+        public ForwardToDeskSettings Settings { get; }
+        
+        private ForwardToDeskAction GetTarget()
+        {
+            return new ForwardToDeskAction(ForwardToDesk, StateManager, Logger);
```

Comment:
The initialization of `ForwardToDeskAction` could be done at `ForwardToDeskActionTests` constructor